mantle/cmd/ore/aliyun/make-public.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aliyun
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdPublic = &cobra.Command{
+		Use:   "make-public <image:region...>",
```

Comment:
optional preference thing:

`region:image` feels more natural to me than the other way around.

Reply:
I debated this back and forth with myself, but I can't really remember what my argument was for this format.  Happy to change it.